frontend/appflowy_flutter/lib/plugins/database/widgets/cell/card_cell_skeleton/text_card_cell.dart

```diff
@@ -181,12 +183,25 @@ class _TextCellState extends State<TextCardCell> {
     );
   }
 
+  ui.TextDirection? lastDirection;
+
   Widget _buildTitle() {
-    final textField = _buildTextField();
     return BlocBuilder<TextCellBloc, TextCellState>(
       builder: (context, state) {
+        final text = _textEditingController.text.isNotEmpty
```

Comment:
How about setting `LocaleKeys.grid_row_titlePlaceholder.tr()` as the hint text instead of the text?